src/tlo/cli.py

```diff
@@ -234,6 +234,25 @@ def batch_submit(ctx, scenario_file, asserts_on, more_memory, keep_pool_alive, i
     task_dir = "${{AZ_BATCH_TASK_DIR}}"
     gzip_pattern_match = "{{txt,log}}"
     command = f"""
+    # Logfile for `earlyoom`
+    EARLYOOM_LOG=/tmp/earlyoom.log
+
+    function terminate_and_log() {{{{
+        # Terminate the `earlyoom` process
+        kill "${{{{EARLYOOM_PID}}}}"
+        # Print line(s) in the log file referencing terminated processes, if any.
+        grep "^sending SIGTERM to process" "${{{{EARLYOOM_LOG}}}}"
+    }}}}
+
+    # Add a trap at exit to terminate `earlyoom` and print relevant log lines.
+    trap terminate_and_log EXIT
+
+    # Delete existing log files, if any.
+    rm -f "${{{{EARLYOOM_LOG}}}}"
+    # Start the `earlyoom` daemon.
+    nohup earlyoom -m 95 -s 100 &> "${{{{EARLYOOM_LOG}}}}" &
```

Comment:
I got the argument `-m 5` backward, the argument is the amount of memory _available_, not _used_:
```suggestion
    # Start the `earlyoom` daemon. Ignore swap usage, terminate process if less than 5% of main memory available.
    nohup earlyoom -m 5 -s 100 &> "${{{{EARLYOOM_LOG}}}}" &
```